docs/build-for-developers/cli-usage.md

```diff
@@ -1,98 +1,137 @@
 ---
 title: Using the CLI
-sidebar_label: CLI Usage
+sidebar_label: Basic Usage
 slug: /cli-usage
 ---
 
-## Prerequisites
+You're probably here to run jobs (expressions) or workflows, which the CLI makes
+easy. Here's an outline of the basic usage scenarios
 
-1. Ensure you have a code editor installed on your machine (e.g.
-   [VS Code](https://code.visualstudio.com/),
-   [Sublime](https://www.sublimetext.com/))
-
-2. Install NodeJs **v18 is the minimum version required**
+### Run a job
 
-   - To install a specific version of Node.js (in this case, version 18) on
-     Linux, Windows, or macOS, you can use a version manager such as nvm (Node
-     Version Manager) or any multiple runtime version manager eg:
-     [asdf](https://github.com/asdf-vm/asdf). These tools allow you to install
-     and switch between multiple versions of Node.js on the same machine. See
-     below for instructions for different operating systems.
-   - Read this article to learn how to install NodeJs in your machine
-     [kinsta.com/blog/how-to-install-node-js/](https://kinsta.com/blog/how-to-install-node-js/)
+To run a single job, you must explicitly specify which adaptor to use.You can
+find the list of publicly available [adaptors here](/adaptors). See examples
+below
 
-3. Have a basic understanding of OpenFn—check out jobs and adaptors, at least,
-   in the [Intro section](documentation/next) of this site.
-4. Install the OpenFn CLI with `npm install -g @openfn/cli`
+> Pass the `-i` flag to auto-install that adaptor (it's safe to do this
+> redundantly).
 
-## CLI Usage - Key Commands
+**Use a shorthand (e.g., `http`)**
```

Comment:
Can we add a colon after the description? Like this: "Use a shorthand (e.g., http):"

Please do this for all of the descriptions that precede a black box